lib/resty/mlcache/ipc.lua

```diff
@@ -137,14 +149,18 @@ function _M:broadcast(channel, data)
 end
 
 
-function _M:poll()
+function _M:poll(max_sleep)
```

Comment:
From a user perspective, I am trying to avoid putting everything in the constructor, and instead only add extra arguments when necessary, on the methods to which they apply, as to give the user some sense of context (from a docs and API POV)